src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateCoverageComponentsInstance.java

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.SparkToggleCommandLineProgram;
+
+import java.io.File;
+
+/**
+ * {@link CalculateCoverageComponents} running class.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+final class CalculateCoverageComponentsInstance extends Main {
```

Comment:
The problem is that I need to run the pca tool in preparation for the test... unfortunately, due to method permission is is rather difficult to reuse gatk4 test framework to execute a tool unless you extend Main or BaseTest with the appropriate  getToolProgramName override... 

I opted for the first but giving a a bit more though it may be better to make CalculateCoverageComponentsIntegrationTest export as public its runCommandLine. This way I avoid to add this additional class for the task.